apps/notifications/src/standalone/auth-wrapper/iframe-proxy.jsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect, useState } from 'react';
+import wpcom from 'wpcom';
+import proxyRequest from 'wpcom-proxy-request';
+
+/**
+ * Internal dependencies
+ */
+import setTracksUser from './set-tracks-user';
+
+const useIframeProxyClient = () => {
+	const [ client, setClient ] = useState( null );
+
+	useEffect( () => {
+		const tempClient = wpcom( proxyRequest );
+		tempClient.request( { metaAPI: { accessAllUsersBlogs: true } }, ( error ) => {
+			if ( error ) {
+				throw error;
```

Comment:
This is a pre-existing bug but: nobody will ever catch this error. It will be thrown out of the callback and ignored.